examples/installation/build.sh

```diff
@@ -55,7 +55,12 @@ xctest() {
     if [[ $NAME == CocoaPods* ]]; then
         pod install --project-directory="$DIRECTORY"
     elif [[ $NAME == Carthage* ]]; then
-        (cd "$DIRECTORY"; carthage update)
+        (
+            cd "$DIRECTORY";
+            : ${sha:=master}
+            echo "github \"realm/realm-cocoa\" \"$sha\"" > Cartfile
```

Comment:
I'd have just used `${sha:-master}` when expanding it into the Cartfile rather than having the separate assign-if-unset, but that's just a matter of style.